packages/ses/test/tame-global-reg-exp-object.test.js

```diff
@@ -5,7 +5,7 @@ import tameGlobalRegExpObject from '../src/tame-global-reg-exp-object.js';
 const { test } = tap;
 
 test('tameGlobalRegExpObject - constructor', t => {
-  t.plan(6);
+  t.plan(8);
```

Comment:
Why do we ***ever*** use `t.plan` rather than `t.end`?

Reply:
I dunno.. it was like that when I found it :). I think JF had an argument for `t.plan` that made sense at the time, but I don't remember what it was.